plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -388,6 +375,25 @@ class NewCreateServiceModal extends Component {
     const {serviceFormErrors, serviceSpec} = this.state;
     let validationErrors = [];
 
+    const APP_VALIDATORS = Hooks.applyFilter(
```

Comment:
I think here it's better to use a filter. Since `doAction` returns `undefined` and I'd rather not mutating anything. I wouldn't care so much that there will be an evil plugin that will add a filter that removes all the validators.